templates/pages/dashboard.html

```diff
@@ -26,19 +26,38 @@
 </div>
 {% endif %}
 
-<p>
-  <a class="btn btn-primary" href="/submit">
-    <i class="fa fa-pen"></i>
-    Add Post
-  </a>
-  <a class="btn btn-secondary" href="/import">
-    <i class="fas fa-file-csv"></i>
-    Import
-  </a>
-  <a class="btn btn-secondary" href="/export">
-    <i class="fas fa-file-csv"></i>
-    Export
-  </a>
-</p>
+<div class="row no-gutters">
```

Comment:
Please remove `no-gutters`. I understand that you would visually prefer to have no padding between the buttons but neither I nor Kris do.
```suggestion
<div class="row">
```

Reply:
I actually agree, it can look good for forms, but it's pretty ugly that the buttons blend into one another. I'll fix this